pom.xml

```diff
@@ -39,6 +39,11 @@
             <version>v1.6.1</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.TheBusyBiscuit</groupId>
+            <artifactId>SlimyTreeTaps</artifactId>
+            <version>b97239fb3a</version>
+        </dependency>
```

Comment:
Why are you even adding this? You are not using any code from SlimyTreeTaps anyway.